README.md

```diff
@@ -50,20 +50,19 @@ Inside each jail, `/app/run` is executed with a working directory of `/app`.
 To configure, [use `ENV`](https://docs.docker.com/engine/reference/builder/#env).
 To remove a limit, set its value to `0`.
 
-| Name                | Default  | Description                                             |
-| ------------------- | -------- | ------------------------------------------------------- |
-| `JAIL_TIME`         | 20       | Maximum wall seconds per connection                     |
-| `JAIL_CONNS`        | 0        | Maximum concurrent connections across all IPs           |
-| `JAIL_CONNS_PER_IP` | 0        | Maximum concurrent connections for each IP              |
-| `JAIL_PIDS`         | 5        | Maximum PIDs per connection                             |
-| `JAIL_MEM`          | 5M       | Maximum memory per connection                           |
-| `JAIL_CPU`          | 100      | Maximum CPU milliseconds per wall second per connection |
-| `JAIL_POW`          | 0        | [Proof of work](#proof-of-work) difficulty              |
-| `JAIL_PORT`         | 5000     | Port number to bind to                                  |
-| `JAIL_SYSCALLS`     | _(none)_ | Additional allowed syscall names separated by `,`       |
-| `JAIL_TMP_SIZE`     | 0        | Maximum size of writable `/tmp` directory               |
+| Name                | Default             | Description                                             |
+| ------------------- | ------------------- | ------------------------------------------------------- |
+| `JAIL_TIME`         | `20`                | Maximum wall seconds per connection                     |
+| `JAIL_CONNS`        | `0`                 | Maximum concurrent connections across all IPs           |
+| `JAIL_CONNS_PER_IP` | `0`                 | Maximum concurrent connections for each IP              |
+| `JAIL_PIDS`         | `5`                 | Maximum PIDs per connection                             |
+| `JAIL_MEM`          | `5M`                | Maximum memory per connection                           |
+| `JAIL_CPU`          | `100`               | Maximum CPU milliseconds per wall second per connection |
+| `JAIL_POW`          | `0`                 | [Proof of work](#proof-of-work) difficulty              |
+| `JAIL_PORT`         | `5000`              | Port number to bind to                                  |
+| `JAIL_DEV`          | `null,zero,urandom` | Device files available in `/dev`                        |
+| `JAIL_SYSCALLS`     | _(none)_            | Additional allowed syscall names separated by `,`       |
+| `JAIL_TMP_SIZE`     | `0`                 | Maximum size of writable `/tmp` directory               |
 
 If it exists, `/jail/hook.sh` is executed before the jail starts.
 Use this script to configure nsjail options or the execution environment.
-
-If `/srv/dev` exists, `/dev/null`, `/dev/zero`, and `/dev/urandom` are available inside the jail.
```

Comment:
This probably still needs to be kept (but modified), since we still don't try to set up `/dev` inside the sandbox if `/srv/dev` [doesn't exist](https://github.com/redpwn/jail/pull/14/commits/d7d477ef03e838068c81a55bafdeee7f4a56cb52#diff-c457573a328590db4aa6f3e6af8ad989163c2c63cc591160de3a5f600fcb2053R36)